packages/sanity/src/desk/comments/src/components/pte/comment-input/Editable.tsx

```diff
@@ -13,17 +13,9 @@ import {renderBlock, renderChild} from '../render'
 import {useCommentInput} from './useCommentInput'
 import {useCursorElement} from './useCursorElement'
 
+const POPOVER_FALLBACK_PLACEMENTS: PopoverProps['fallbackPlacements'] = ['bottom', 'top']
 const INLINE_STYLE: React.CSSProperties = {outline: 'none'}
-
-const EditableWrapStack = styled(Stack)(() => {
-  return css`
-    & > div:first-child {
```

Comment:
This CSS is not used anymore since a comment can only contain one block. This CSS was previously used to add spacing between individual blocks in the editor.